tests/plugins/test_topic_checking.py

```diff
@@ -0,0 +1,442 @@
+# Copyright (c) 2015 Nicolas JOUANIN
```

Comment:
this file was neither created in 2015, nor by nicolas.

please remove this line :)

Reply:
Heh, I did a copy from `test_authentication.py`… but I suppose I've thrown out enough for it to not be considered a "derivative work".  Question is, what should appear in the header?

Something like this?
```
# Copyright (c) 2021 aMQTT Contributers (https://github.com/Yakifo/amqtt/graphs/contributors)
# SPDX-License-Identifier: MIT
```

Reply:
IANAL - so this is no legal advice just what I found online and my own understanding of navigating copyright laws:

There is no legal reason to have these lines at all. Therefore I would actually prefer not have them at all.  We do not have to remind the reader of the code in every file what the license is...

But out of respect (and not legal requirement) to the original owner of hbmqtt I would not go as far as to remove them all over the code base as it is really close to what he wrote.

Reply:
@HerrMuellerluedenscheid your take?

Reply:
Yes, agreed. Please omit these on newly added files.